src/Symfony/Contracts/Cache/composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.2",
```

Comment:
If contracts don't get a new major version (you are not bumping to 4.0-dev), should we really bump their min PHP version ?